settings/installer/ospd/ospd.spec

```diff
@@ -0,0 +1,119 @@
+---
+subparsers:
+    ospd:
+        help: Installs openstack using OSP Director
+        groups:
+            - title: Firewall
+              options:
+                firewall-rules:
+                    type: YamlFile
+                    help: The list of firewall rules
+                    default: default.yml
+
+            - title: Product
+              options:
+                product:
+                    type: YamlFile
+                    help: The product to install
+                    default: "rhos-latest.yml"
+
+            - title: Undercloud
+              options:
+                undercloud-network:
+                    type: YamlFile
+                    help: The undercloud network config
+                    default: default.yml
+
+            - title: Overcloud
+              options:
+                overcloud-use_ssl:
```

Comment:
please don't use "underscore" (_) in flag names. 
This should simply be: 
overcloud-ssl